docs/rules/no-unused-modules.md

```diff
@@ -0,0 +1,80 @@
+# import/no-unused-modules
+
+Reports:
+  - modules without any or
+  - individual exports not being used within other modules
+
+## Rule Details
+
+
+### Options
+
+This rule takes the following option:
+
+- `src`: an array with files/paths to be analyzed. It only for applies for unused exports  
```

Comment:
i'm confused about this one - rule schemas don't generally specify their own paths, eslint itself does.

Reply:
For my knowledge, eslint does not share this information with the rules - but I need it for building up a list of existing exports and imports for this rule to work.

If there is a way to get this information from eslint, I would actually be glad to get rid of this option.

Reply:
I'm confused why this wouldn't just use "the whole project" - ie, `process.cwd()` recursively.

Reply:
In my opinion, both solutions aren't optimal...

If we would use `process.cwd()`, the rule would always start its analysis within the root directory, even if eslint is restricted to a single directory within the root directory. In this case, one could of course use the `ignore`-option to restrict the files being linted...but then one could as well just maintain the `src`-option providing the one directory he intends to lint.

Reply:
Good idea. @ljharb: what do you think about this?

Reply:
that seems more reasonable.

Reply:
I added the default src with aa7253baf383f370b3dccefbb11b17cd5db2d8dc. Readme will be updated later on.